bulk-upload-resources-python-example/upload_resources.py

```diff
@@ -19,20 +19,20 @@
 config.read(config_path)
 
 
-def get_auth(config):
+def get_auth(config: RawConfigParser):
     """ Get the authentication out of our config file """
     return config.get("domain", "client-key"), config.get("domain", "client-secret")
 
 
-def convert_datetime_to_epoch(datetime_string):
+def convert_datetime_to_epoch(datetime_string: str):
     """ Converts the datetime to epoch in miliseconds there are two possible time formats"""
     try:
         return datetime.strptime(datetime_string, '%Y-%m-%dT%H:%M:%SZ').timestamp() * 1000
     except:
         return datetime.strptime(datetime_string, '%Y-%m-%dT%H:%M:%S.%fZ').timestamp() * 1000
 
 
-def transform_data(data):
+def transform_data(data: pandas.DataFrame):
     """Transforms data from the csv format to the expected format bij the api"""
```

Comment:
Again consistency is important, some comments start out with a space before and after the sting, best to chose one style `"""A comment"""` or `""" A comment """` are both good.